src/dstack/_internal/core/backends/aws/resources.py

```diff
@@ -596,3 +602,35 @@ def _is_private_subnet_with_internet_egress(
                     return True
 
     return False
+
+
+def get_reservation(
+    ec2_client: botocore.client.BaseClient,
+    reservation_id: str,
+    instance_count: Optional[int] = 0,
+    instance_types: Optional[List[str]] = None,
+    is_capacity_block: Optional[bool] = False,
+) -> Optional[str]:
```

Comment:
I believe it's not an str that's returned?

Reply:
Fixed in [d48d3ec](https://github.com/dstackai/dstack/pull/1977/commits/d48d3ecefcf08c2e5d9da6a516222e512e666f03)